test/test_decision.py

```diff
@@ -91,6 +92,55 @@ def test_no_email_owner(self):
         params = decision.get_decision_parameters(FAKE_GRAPH_CONFIG, self.options)
         self.assertEqual(params["owner"], "ffxbld@noreply.mozilla.org")
 
+    @unittest.mock.patch.object(
+        GitRepository,
+        "get_commit_message",
+        unittest.mock.MagicMock(return_value="Add Foo"),
+    )
+    @unittest.mock.patch.object(
+        HgRepository,
+        "get_commit_message",
+        unittest.mock.MagicMock(return_value="Add Foo"),
+    )
+    def test_default(self):
```

Comment:
Nit: Let's improve the test name with a more explicit name 🙂 

```suggestion
    def test_regular_commit_message_yields_default_target_tasks_method(self):
```